ui/modal/create_watch_only_modal.go

```diff
@@ -110,6 +110,11 @@ func (cm *CreateWatchOnlyModal) Handle() {
 			cm.Dismiss()
 		}
 	}
+
+	if cm.modal.BackdropClicked() {
```

Comment:
A check should be made before closing to avoid closing the modal while a process is running.